ci-operator/step-registry/ipi/conf/alibabacloud/ipi-conf-alibabacloud-commands.sh

```diff
@@ -33,3 +33,7 @@ platform:
     region: ${REGION}
     resourceGroupID: ${CI_RESOURCE_GROUP_ID}
 EOF
+
+if [[ -s "${SHARED_DIR}/customer_vpc_subnets.yaml" ]]; then
```

Comment:
Suggest to introduce a dedicated step to do the customer vpc setting, similar to https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/private/provision/cucushift-installer-rehearse-azure-ipi-private-provision-chain.yaml#L9

Reply:
I meant we can introduce `ipi-conf-alibabacloud-provisionednetwork` step to test the customer vpc case.